bigtable/internal/testproxy/proxy.go

```diff
@@ -102,10 +102,10 @@ func (c credentialsBundle) NewWithMode(mode string) (credentials.Bundle, error)
 //
 // There are three base cases to address:
 //  1. CreateClientRequest specifies no unique credentials; so ADC will be used.
-//     This method returns an empty slice.
-//  2. CreateClientRequest specifies both call and channel credentials. In
-//     this case, we need to create a combined credential.
-//  3. CreateClientRequest specifies only a channel credential.
+//     This method returns an empty slice..
+//  2. CreateClientRequest specifies only a channel credential.
+//  3. CreateClientRequest specifies both call and channel credentials. In
```

Comment:
Does the ordering matter? Also the docstring doesn't agree with the code in terms of order.

Reply:
No, the ordering doesn't matter. This comment provides guidance for future maintainers of this code, by giving explicit numbers to each branch/technique for getting credentials.